assets/src/dashboard/app/views/myStories/index.js

```diff
@@ -94,18 +76,29 @@ function MyStories() {
     }
   }, [viewStyle]);
 
+  const filteredStoriesCount = filteredStories.length;
+  const hasFilteredStories = filteredStoriesCount > 0;
```

Comment:
Nit, it looks like `filteredStoriesCount` sole purpose is to help define `hasFilteredStories`, couldn't `hasFilteredStories` be defined as the following?
```
const hasFilteredStories = filteredStories.length > 0;
```
And then you won't need `filteredStoriesCount`.

Reply:
I'm using the count in the label below this, that's why i separated them.

Reply:
whoops! yup it's supposed to be used below it but it wasn't. let me fix this...